libr/io/p/io_ihex.c

```diff
@@ -436,7 +437,7 @@ RIOPlugin r_io_plugin_ihex = {
 #ifndef CORELIB
 R_API RLibStruct radare_plugin = {
 	.type = R_LIB_TYPE_IO,
-	.data = &r_io_plugin_hex,
+	.data = &r_io_plugin_ihex,
```

Comment:
There was no `&r_io_plugin_hex`, it was giving me an error. So I changed this as well.